base/strings/util.jl

```diff
@@ -58,21 +58,20 @@ startswith(a::Vector{UInt8}, b::Vector{UInt8}) =
 
 # TODO: fast endswith
 
-
 """
     chop(s::AbstractString)
 
 Remove the last character from `s`.
 
 ```jldoctest
-julia> a = string("March")
+julia> a = "March"
 "March"
 
 julia> chop(a)
 "Marc"
 ```
 """
-chop(s::AbstractString) = s[1:end-1]
+chop(s::AbstractString) = SubString(s, 1, endof(s)-1)
```

Comment:
It looks to me like most of the failures are due to line numbers being not the same as expected?

https://gist.github.com/TotalVerb/d81e7fff617d315f51ecd019aac4f553